### PR TITLE
doc: silence sphinx warnings

### PR DESCRIPTION
**Pull request by tchaikov**

### CODE REVIEW

File: doc/cephfs/troubleshooting.rst [hunk 1]
```diff
@@ -1,22 +1,26 @@
 =================
  Troubleshooting
 =================
+
 Slow/stuck operations
-~~~~~~~~~~~~~~~~
+=====================
```
Comment (tchaikov) [inline doc/cephfs/troubleshooting.rst]: this addresses ```SEVERE: Title level inconsistent:```

**Comment by tchaikov** [comment]
@badone could you help me review this change?

File: doc/radosgw/swift.rst [hunk 2]
```diff
@@ -67,7 +67,7 @@ The following table describes the support status for current Swift functional fe
 +---------------------------------+-----------------+----------------------------------------+
 | **Expiring Objects**            | Supported       |                                        |
 +---------------------------------+-----------------+----------------------------------------+
-| **Object Versioning**           | Supported   |                                        |
+| **Object Versioning**           | Supported       |                                        |
```
Comment (tchaikov) [inline doc/radosgw/swift.rst]: see http://docs.ceph.com/docs/master/radosgw/swift/

**Comment by badone** [comment]
lgtm

**Pull request merged**